roles/tempest/tasks/create-clouds-file.yml

```diff
@@ -36,3 +36,4 @@
   ansible.builtin.template:
     src: clouds.yaml.j2
     dest: "{{ cifmw_tempest_artifacts_basedir }}/clouds.yaml"
+    mode: "0775"
```

Comment:
should it be 0644 like all other files here?

Reply:
+1